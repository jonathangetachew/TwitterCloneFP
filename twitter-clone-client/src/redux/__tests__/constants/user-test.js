import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import userConstants from '../../constants/user';

describe('UserConstants', () => {
    jest.useFakeTimers();

    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it(`render constants`, () => {
        expect(userConstants).toMatchSnapshot();
    });
});