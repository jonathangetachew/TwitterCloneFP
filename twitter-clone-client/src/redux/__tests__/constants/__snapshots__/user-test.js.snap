// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserConstants render constants 1`] = `
Object {
  "LOGIN": "LOGIN",
  "LOGIN_FAIL": "LOGIN_FAIL",
  "LOGIN_REQUEST": "LOGIN_REQUEST",
  "LOGIN_SUCCESS": "LOGIN_SUCCESS",
  "LOGOUT": "LOGOUT",
  "LOGOUT_REQUEST": "LOGOUT_REQUEST",
  "REGISTER": "REGISTER",
  "REGISTER_FAIL": "REGISTER_FAIL",
  "REGISTER_REQUEST": "REGISTER_REQUEST",
  "REGISTER_SUCCESS": "REGISTER_SUCCESS",
  "UPDATE": "UPDATE",
  "UPDATE_FAIL": "UPDATE_FAIL",
  "UPDATE_REQUEST": "UPDATE_REQUEST",
  "UPDATE_SUCCESS": "UPDATE_SUCCESS",
}
`;
